stdlib/public/core/Runtime.swift.gyb

```diff
@@ -247,12 +247,11 @@ public // @testable
 func _stdlib_atomicLoadARCRef(
   object target: UnsafeMutablePointer<AnyObject?>
 ) -> AnyObject? {
-  let result = _swift_stdlib_atomicLoadPtrImpl(
-    object: UnsafeMutablePointer(target))
-  if result != nil {
-    return Unmanaged<AnyObject>.fromOpaque(result).takeUnretainedValue()
+  guard let result = _swift_stdlib_atomicLoadPtrImpl(
+  object: UnsafeMutablePointer(target)) else {
```

Comment:
Hoe di find these hackers, Stolen data evidence to report to The Police 


Eniro

Den 7. des. 2015 kl. 20.24 skrev Dmitri Gribenko <notifications@github.com>:

In stdlib/public/core/Runtime.swift.gyb:

> @@ -247,12 +247,11 @@ public // @testable
>  func _stdlib_atomicLoadARCRef(
>    object target: UnsafeMutablePointer<AnyObject?>
>  ) -> AnyObject? {
> -  let result = _swift_stdlib_atomicLoadPtrImpl(
> -    object: UnsafeMutablePointer(target))
> -  if result != nil {
> -    return Unmanaged<AnyObject>.fromOpaque(result).takeUnretainedValue()
> +  guard let result = _swift_stdlib_atomicLoadPtrImpl(
> +  object: UnsafeMutablePointer(target)) else {
This part LGTM, but the line continuation should be indented 2 spaces.

—
Reply to this email directly or view it on GitHub.


-- 
Confidentiality Notice: This e-mail transmission may contain confidential 
or legally privileged information that is intended only for the individual 
or entity named in the e-mail address. If you are not the intended 
recipient, you are hereby notified that any disclosure, copying, 
distribution, or reliance upon the contents of this e-mail is strictly 
prohibited and may be unlawful. If you have received this e-mail in error, 
please notify the sender immediately by return e-mail and delete all copies 
of this message.